layouts/_default/events.yaml

```
@@ -0,0 +1,18 @@
+{{- $events := .Site.GetPage "events" -}}
+---
+events:
+{{- range sort $events.Pages ".Date" "asc" -}}
+{{- $images := .Resources.ByType "image" }}
+{{- $feature := $images.GetMatch .Params.feature }}
+- title: {{ .Params.title }}
+  date: {{ .Date.Format "2006-01-02T15:04:05Z07:00" }}
```

I'll need to double check if this works, but for consistency this is what we're using and works already
```suggestion
  date: {{ .Date.Format "2019-06-20 18:45:00 +0200" }}
```